app/queues/notifications/certificate-expiration-monitor-worker.server.ts

```diff
@@ -0,0 +1,53 @@
+import { Worker } from 'bullmq';
+import { redis } from '~/lib/redis.server';
+import logger from '~/lib/logger.server';
+import type { DomainStatusData } from './record-cert-expiration-data.interface';
+import { addNotification } from './notifications.server';
+import { prisma } from '~/db.server';
+
+// name for the queue
+export const certExpirationQueueName = 'notifications-certExpiration';
+const updateCertificateNotifiedStatus = async (certId: number) => {
+  return await prisma.certificate.update({
```

Comment:
in this case async too :)
- Await will happen automatically at return
- since you are returning a promise from prisma.certificate.update, no need to do async, you'll get a promise either way